uisdk/src/test/java/com/karhoo/uisdk/screen/rides/detail/RideDetailPresenterTest.kt

```diff
@@ -614,7 +614,7 @@ class RideDetailPresenterTest {
     @Test
     fun `When the trip has a service cancellation of type before driver en route with a confirmed status, the cancellation text is shown`() {
         presenter.checkCancellationSLA(
-                TripStatus.CONFIRMED,
+                TripInfo(),
```

Comment:
You need to set the TripStatus to `TripStatus.CONFIRMED` ... same for other tests in this file. We need to keep the expected logic

Reply:
checkCancellationSLA no longer accepts a trip status though, I haven't found a way to pass this through otherwise since it wants the TripInfo object not the TripStatus object.